R/zchunk_L100.Population_downscale_ctry.R

```diff
@@ -34,19 +34,129 @@ module_socioeconomics_L100.Population_downscale_ctry_DISABLED <- function(comman
     UN_popTot <- get_data(all_data, "socioeconomics/UN_popTot")
 
     # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
-    #
+
+    ## (1) Historical population by country
+
+    # First clean up Maddison raw data -- NOTE: Maddison data are used to develop population ratios relative to 1950 to combine with UN data from 1950 onward
+    pop_thous_ctry_reg <- Maddison_population %>%
+      filter(!is.na(Country)) %>%  # Remove values with missing country name (all year values are also missing for these--probably just blank rows in the original data)
```

Comment:
This really should have been taken care of in the chunk that generated the `Maddison_populaiton` table.  If this is the only place that data is used (i.e., downstream modules use it only indirectly through the outputs of this module), then I'd say let it be.  However, if any other modules are using that table, then we should fix it at its source.